polynote-server/src/main/scala/polynote/server/SocketSession.scala

```diff
@@ -158,22 +161,20 @@ class SocketSession(
 
     case RunCell(path, ids) =>
       getNotebook(path).flatMap {
-        notebookRef => notebookRef.runCells(ids)
+        notebookRef => notebookRef.runCells(ids).map(_.drain) // we throw away the messages because they already come through the notebook's messages stream
```

Comment:
This is kind of confusing... should try to express more clearly why we are throwing the messages from the resulting Stream on the ground. (Oblig: [happy birthday to the ground!](https://www.youtube.com/watch?v=gAYL5H46QnQ))

Reply:
Is it because those messages aren't my dad?

Reply:
No, it's because they're not a part of your system.